app/views/admin/mentoring/_offices_mentoring.html.erb

```diff
@@ -0,0 +1,23 @@
+<table cellspacing="0" cellpadding="0" class="mentoring_table">
+  <thead>
+    <tr>
+      <th><%= I18n.t('offices') %></th>
+      <th><%= I18n.t('mentors') %></th>
+      <th><%= I18n.t('mentees') %></th>
+    </tr>
+  </thead>
+  <tbody>
+    <% city_name = nil %>
+    <% mentorings.each do |mentoring| %>
+      <% city = mentoring.city == city_name ? nil : mentoring.city %>
+      <% city_name = mentoring.city %>
```

Comment:
sorry not sure if I understood correctly, are you trying to group the mentors by city in here?

Reply:
Basically yes and also to remove the repeated city names.